doc/ug/tutorial.ThermodynamicIntegration.rst

```diff
@@ -0,0 +1,157 @@
+Thermodynamic integration
+=========================
+
+.. |espp| replace:: ESPResSo++
+
+Theoretical explanation
+-----------------------
+
+Thermodynamic integration (TI) is a method used to calculate the free energy difference between two states A and B. For the theoretical background, see e.g. http://www.alchemistry.org. In this tutorial, we show how to perform TI calculations with ESPResSo++. We calculate the free energy of solvation of methanol in water. The complete python script is available in the ESPResSo+ source code under ``$ESPRESSOHOME/examples/thd_integration_solvation`` 
```

Comment:
It's supposed to be the espressopp root directory. Actually if I just write "available in the ESPResSo+ source code under examples/thd_integration_solvation" I guess that's clear.